pkg/porter/build.go

```diff
@@ -338,6 +339,33 @@ func (p Porter) writeBundle(b bundle.Bundle) error {
 	return errors.Wrap(err, "error writing to cnab/bundle.json")
 }
 
+func (p *Porter) generateBundleImages() map[string]bundle.Image {
+	images := make(map[string]bundle.Image, len(p.Manifest.ImageMap))
+
+	for i, refImage := range p.Manifest.ImageMap {
+		img := bundle.Image{
+			Description: refImage.Description,
+			BaseImage: bundle.BaseImage{
```

Comment:
Q: I see we are re-using `bundle.Image` and `BaseImage` structs to build up a corresponding mapped image.  Should we be using the `MappedImage` struct added to `manifest.go` instead?

Reply:
This is going to be more clear in my next pull request #424 where I've split out the code for mapping a porter manifest to a bundle.json (I hope!). This code right here is making a bundle.json, so we are using the bundle structs from cnab-go. MappedImage is a struct for working with the porter manifest.

Reply:
Ah!  I've got it now.  Thank you for spelling it out for me.